tests/bsim/bluetooth/audio/src/csip_set_coordinator_test.c

```diff
@@ -110,6 +112,13 @@ static void csip_lock_changed_cb(struct bt_csip_set_coordinator_csis_inst *inst,
 	printk("Inst %p %s\n", inst, locked ? "locked" : "released");
 }
 
+static void csip_sirk_changed_cb(struct bt_csip_set_coordinator_csis_inst *inst)
```

Comment:
Isn´t this only checking that we receive sirk_changed once? I.e. we are setting the flag on the first notification?
Correct me if I'm wrong - but if I interpret this right, shouldnt we wait until all connections have sent the notify?

Reply:
> Isn´t this only checking that we receive sirk_changed once? I.e. we are setting the flag on the first notification?

> Correct me if I'm wrong - but if I interpret this right, shouldnt we wait until all connections have sent the notify?

Arguably yes. It doesn't really change the behavior of the test as they are running the same software: If one works, they should all work. If you prefer I can modify this to expect `N` number of notifications.

Reply:
I would prefer this yes, even if you certainly are technically correct, I think that this would make the test look more logical and easier to interpret as to what should happen.

Reply:
@fredrikdanebjer Mind if we do that in a follow up? I'd rather try to get this into the release now, rather than postponing it due to a test update :)

Reply:
@Thalley that is fair enough